site/content/api/Device-Connection-API.md

```diff
@@ -0,0 +1,145 @@
+
+Implementors of this API may return a *404* status code in order to indicate that no device with the given identifier exists for the given tenant. However, performing such a check is optional.
+
+For status codes indicating an error (codes in the `400 - 499` range) the message body MAY contain a detailed description of the error that occurred.
+
+## Standard Message Properties
+
+Due to the nature of the request/response message pattern of the operations of the Device Connection API, there are some standard properties shared by all of the request and response messages exchanged as part of the operations.
+
+### Standard Request Properties
+
+The following table provides an overview of the properties shared by all request messages regardless of the particular operation being invoked.
+
+| Name             | Mandatory | Location                 | AMQP Type    | Description |
+| :--------------- | :-------: | :----------------------- | :----------- | :---------- |
+| *subject*        | yes       | *properties*             | *string*     | MUST be set to the value defined by the particular operation being invoked. |
+| *correlation-id* | no        | *properties*             | *message-id* | MAY contain an ID used to correlate a response message to the original request. If set, it is used as the *correlation-id* property in the response, otherwise the value of the *message-id* property is used. |
+| *message-id*     | yes       | *properties*             | *string*     | MUST contain an identifier that uniquely identifies the message at the sender side. |
+| *reply-to*       | yes       | *properties*             | *string*     | MUST contain the source address that the client wants to received response messages from. This address MUST be the same as the source address used for establishing the client's receive link (see [Preconditions]({{< relref "#preconditions" >}})). |
+| *device_id*      | yes       | *application-properties* | *string*     | MUST contain the ID of the device that is subject to the operation. |
+
+### Standard Response Properties
+
+The following table provides an overview of the properties shared by all response messages regardless of the particular operation being invoked.
+
+| Name             | Mandatory | Location                 | AMQP Type    | Description |
+| :--------------- | :-------: | :----------------------- | :----------- | :---------- |
+| *correlation-id* | yes       | *properties*             | *message-id* | Contains the *message-id* (or the *correlation-id*, if specified) of the request message that this message is the response to. |
+| *content-type*   | yes       | *properties*             | *string*     | MUST be set to `application/json` if the response message's status code is 200 and the body of the message contains the result of invoking the operation. |
+| *device_id*      | yes       | *application-properties* | *string*     | Contains the ID of the device. |
```

Comment:
IMHO we do not need this, do we?

Reply:
Same argument as above, shouldn't we keep the set of standard response properties somewhat consistent?

Reply:
Just read your comment above - I've removed it accordingly.